lua/snacks/git.lua

```diff
@@ -14,13 +14,17 @@ Snacks.config.style("blame_line", {
   ft = "git",
 })
 
-local git_cache = {} ---@type table<string, boolean>
+local git_cache = {} ---@type table<string, string>
 local function is_git_root(dir)
-  if git_cache[dir] == nil then
-    git_cache[dir] = (vim.uv or vim.loop).fs_stat(dir .. "/.git") ~= nil
+  if git_cache[dir] == nil and (vim.uv or vim.loop).fs_stat(dir .. "/.git") ~= nil then
+    git_cache[dir] = vim.fs.normalize(dir)
   end
   return git_cache[dir]
 end
+local function get_git_root()
```

Comment:
vim.system is not available on 0.9, so we can't use that.

Reply:
replaced with `vim.fn.system("git rev-parse --show-toplevel")`, which should be supported by 0.9 iirc?